src/media/css/elements--categories.styl

```diff
@@ -0,0 +1,115 @@
+@import 'lib';
+
+// Originally from elements--nav.styl.
+$color--nav--child-bg = #1A95FC;
+
+mkt-category-list {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    list-style: none;
+    width: 100%;
+
+    +media--wide-tablet() {
+        flex-direction: row;
+        justify-content: center;
+    }
+}
+
+mkt-category-item {
+    width: 100%;
+
+    +media--wide-tablet() {
+        margin: 8px;
+        width: auto;
+    }
+}
+
+.mkt-category-link {
```

Comment:
Should this be `.mkt-category-item--link` instead? I'm not totally up on the naming scheme here but that seems like it would be accurate but a little verbose.